src/inc/til/color.h

```diff
@@ -156,6 +156,22 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             wss << std::setw(2) << static_cast<int>(g);
             wss << std::setw(2) << static_cast<int>(b);
 
+            return wss.str();
+        }
+        std::wstring ToHexString(const bool omitAlpha = false) const

```

Comment:
1. the name on this is weird
2. can we deduplicate the code between to_string and ToHexString?

Eventually I'll come through and replace these with fmt::formatter specializations. :smile:

Reply:
1. one could argue the `to_string` name is the weird one 😉 
2. de-dup'd

Reply:
Not blocking, but we do also have `with_alpha` in snake_case ;P